### PR TITLE
[OPENENGSB-2946] Changed to ProcessBuilder, redirected ErrorStream and u...

### PR DESCRIPTION
**Pull request by kristofmeixner**
...sed command array

### CODE REVIEW

File: wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java [hunk 1]
```diff
@@ -55,7 +54,8 @@
      */
     private String wsdl_location;
     /**
-     * Namespace of the WSDL file. This should be the namespace in which a domain should be located.
+     * Namespace of the WSDL file. This should be the namespace in which a
```
Comment (anpieber) [inline wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java:57]: I would say the deafult openengsb formatter will create 120 chars per line for comments; not 80

File: wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java [hunk 2]
```diff
@@ -239,8 +244,8 @@ private void executeACommand(Process child) throws IOException, MojoExecutionExc
     }
 
     /**
-     * Search in the default folder location of SDK and the .net Framework for the newest version. If the folder exist,
-     * add it to the list
+     * Search in the default folder location of SDK and the .net Framework for
```
Comment (anpieber) [inline wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java:247]: I would say the deafult openengsb formatter will create 120 chars per line for comments; not 80

File: wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java [hunk 3]
```diff
@@ -220,7 +224,8 @@ private void executeACommand(Process child) throws IOException, MojoExecutionExc
             input += tmp + "\n";
             last = tmp;
         }
-        // Because the wsdl.exe can not be executed in a outputDirectory, the file has to be moved to the corresponding
+        // Because the wsdl.exe can not be executed in a outputDirectory, the
```
Comment (anpieber) [inline wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java:227]: I would say the deafult openengsb formatter will create 120 chars per line for comments; not 80

**Comment by anpieber** [comment]
a) sorry for my delay, seams I've missed the notification about this one
b) except for the minor formatting issues it looks pretty good.

File: wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java (hunk 3 above)
Comment (kristofmeixner) [inline wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java:227]: I don´t think so and already wondered myself. Downloaded just seconds ago from http://dev.openengsb.org/resources/eclipse/formatter.xml - setting id="org.eclipse.jdt.core.formatter.comment.line_length" value="80"

Comment (anpieber) [inline wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java:227]: OK, my mistake then. Thx and I'll merge

**Pull request merged**